language/libra-vm/src/errors.rs

```diff
@@ -78,19 +96,30 @@ pub fn convert_normal_prologue_error(status: VMStatus) -> VMStatus {
 /// `UNEXPECTED_ERROR_FROM_KNOWN_MOVE_FUNCTION`
 pub fn convert_normal_success_epilogue_error(status: VMStatus) -> VMStatus {
```

Comment:
I think we should drop the "success" from the name of this function: "convert_normal_epilogue_error"

Reply:
The flows are still separate in the VM, i.e. there still is a success and failure epilogue control flow. Each one of those expects different errors, so I'll keep the name same for the error handling